components/ActionSheet/examples/ActionSheetTypicalUse.m

```diff
@@ -0,0 +1,106 @@
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "ActionSheetTypicalUse.h"
+
+#import "MaterialButtons.h"
+#import "MaterialButtons+ButtonThemer.h"
+
+@interface ActionSheetTypicalUse ()
+
+@property(nonatomic, strong) MDCButton *showButton;
+
+@end
+
+@implementation ActionSheetTypicalUse {
+  MDCButtonScheme *_buttonScheme;
+}
+
+- (instancetype)init {
+  self = [super init];
+  if (self) {
+    self.title = @"Action Sheet";
+    _colorScheme = [[MDCSemanticColorScheme alloc] init];
+    _typographyScheme = [[MDCTypographyScheme alloc] init];
+    _showButton = [[MDCButton alloc] init];
+    _buttonScheme = [[MDCButtonScheme alloc] init];
+  }
+  return self;
+}
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+  
+  self.view.backgroundColor = _colorScheme.backgroundColor;
+  [_showButton setTitle:@"Show action sheet" forState:UIControlStateNormal];
+  [_showButton sizeToFit];
+  _buttonScheme.colorScheme = _colorScheme;
+  _buttonScheme.typographyScheme = _typographyScheme;
+  [MDCContainedButtonThemer applyScheme:_buttonScheme toButton:_showButton];
+  [_showButton addTarget:self
+                  action:@selector(showActionSheet)
+        forControlEvents:UIControlEventTouchUpInside];
+  [self.view addSubview:_showButton];
+}
+
+- (void)viewDidLayoutSubviews {
+  [super viewDidLayoutSubviews];
+
+  _showButton.center = CGPointMake(self.view.center.x, self.view.center.y - 80);
+}
+
+
+
+- (void)showActionSheet {
+  MDCActionSheetController *actionSheet = [[MDCActionSheetController alloc] init];
+  MDCActionSheetAction *homeAction = [MDCActionSheetAction actionWithTitle:@"Home"
+                                                                 image:[UIImage imageNamed:@"Home"]
+                                                               handler:nil];
+  MDCActionSheetAction *favoriteAction =
+      [MDCActionSheetAction actionWithTitle:@"Favorite"
+                                      image:[UIImage imageNamed:@"Favorite"]
+                                    handler:^(MDCActionSheetAction *action){
+                                      NSLog(@"Favorite Action");
+                                    }];
+  MDCActionSheetAction *emailAction =
+      [MDCActionSheetAction actionWithTitle:@"Email"
+                                      image:[UIImage imageNamed:@"Email"]
+                                    handler:^(MDCActionSheetAction *action){
+                                      NSLog(@"Email Action");
+                                    }];
+  [actionSheet addAction:homeAction];
+  [actionSheet addAction:favoriteAction];
+  [actionSheet addAction:emailAction];
+  [self presentViewController:actionSheet animated:YES completion:nil];
+}
+
+@end
+
+@implementation ActionSheetTypicalUse (CatalogByConvention)
+
++ (NSArray *)catalogBreadcrumbs {
+  return @[ @"Action Sheet", @"Action Sheet"];
+}
+
++ (BOOL)catalogIsPresentable {
```

Comment:
Why is this an "Alpha" component if it's presentable and shipping in our Catalog?

Reply:
Opened #4995 to fix this.